bridge-stp.conf

```diff
@@ -1,7 +1,7 @@
 # If 'y' or commented out, mstpd will be automatically started/stopped as
 # needed.  If 'n', mstpd will not be automatically started/stopped by the
 # bridge-stp script.
-MANAGE_MSTPD='n'
+#MANAGE_MSTPD='n'
```

Comment:
91c71b1 adds new functionality but doesn't change the default behavior.  This commit changes the default behavior.  I separated it out in case we wanted to skip changing the default behavior, or in case we wanted to revert the default later.  But I can merge it with 91c71b1 if you want.